libs/backend/src/cast_vote_records/build_cast_vote_record.test.ts

```diff
@@ -760,7 +761,7 @@ test('buildCastVoteRecord - HMPB ballot with unmarked write-in', () => {
   });
 });
 
-describe('hash manipulation', () => {
+test('hash manipulation', () => {
```

Comment:
This was actually incorrect before. The assertions were running outside the context of a test.